user-guide/Advanced_Modules/Dashboards_app/Using_dashboard_feeds/Configuring_GQI_feeds/Configuring_an_external_data_source_in_a_query.md

```diff
@@ -47,6 +47,26 @@ Depending on how the script is configured, there can be additional configuration
 
 An ad hoc data source is represented as a class that implements predefined interfaces. The interfaces you can use are detailed below.
 
+From DataMiner 10.3.4/10.4.0 onwards, ad hoc data sources can retrieve data by means of DMS messages. <!-- RN 35701 -->
```

Comment:
It doesn't make much sense to me to add this info (up to line 69) as part of the introduction about the interfaces in the ad hoc data script, while it's actually a procedure in itself. I think it would be better to add an extra subsection, e.g. "## Retrieving data by means of DMS messages" and to add this procedure in there.